tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -13,6 +13,10 @@ ${RHOSAK_DISPLAYED_APPNAME}=    OpenShift Streams for Apache Kafka
 
 
 *** Test Cases ***
+Verify That Login Page Is Shown When Reaching The RHODS Page
+    [Tags]  ODS-694
```

Comment:
i think u missed Sanity
[Tags]  Sanity
    ...     ODS-531

else looks good to me

Reply:
I added Sanity.